Dalamud/Game/ClientState/Buddy/BuddyList.cs

```diff
@@ -16,7 +17,10 @@ namespace Dalamud.Game.ClientState.Buddy;
 [PluginInterface]
 [InterfaceVersion("1.0")]
 [ServiceManager.BlockingEarlyLoadedService]
-public sealed partial class BuddyList : IServiceType
+#pragma warning disable SA1015
```

Comment:
This is fixed in https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3641, we should pull in the latest prerelease version of the analyzer. Not the responsibility of this PR, though.